skorch/tests/test_probabilistic.py

```diff
@@ -114,6 +114,21 @@ def forward(self, x):
         return latent_pred
 
 
+class MyBernoulliLikelihood(gpytorch.likelihoods.BernoulliLikelihood):
+    """This class only exists to add a param to BernoulliLikelihood
+
+    BernoulliLikelihood used to have parameters before gpytorch v1.10, but now
+    it does not have any parameters anymore. This is not an issue per se, but
+    there are a few things we cannot test anymore, e.g. that parameters are
+    passed to the likelihood correctly when using grid search. Therefore, create
+    a custom class with a (pointless) parameter.
+
+    """
+    def __init__(self, *args, max_plate_nesting=1, **kwargs):
+        self.max_plate_nesting = max_plate_nesting
```

Comment:
Given that the parameter is not used. Can this name be changed to something that is meaningless in the context of Gaussian Processes?

```suggestion
    def __init__(self, *args, search_parameter=1, **kwargs):
        self.search_parameter = search_parameter
```

Reply:
I renamed the parameter to `some_parameter`. At first, I thought it should not work because the same test is re-used for the 2 other classes of GP that use a different likelihood. But it turns out that passing a non-existing parameter does not cause an error:

https://github.com/cornellius-gp/gpytorch/blob/d23099674f67f04f40467964c37fc02f2e2a8dda/gpytorch/likelihoods/gaussian_likelihood.py#L104-L114

It is a bit strange, thus I put a comment there, but I guess for the purpose of the test we can live with that.